provider/marathon/marathon_test.go

```diff
@@ -99,6 +99,7 @@ func TestMarathonLoadConfig(t *testing.T) {
 				`frontend-test`: {
 					Backend:        "backend-test",
 					PassHostHeader: true,
+					BasicAuth:      []string{},
```

Comment:
Yes its required; all of the tests failed given how they just blind compare the structs.